specification/resources/resource-manager/Microsoft.Resources/stable/2021-04-01/resources.json

```
@@ -4568,7 +4568,7 @@
           "description": "The URI of the template. Use either the templateLink property or the template property, but not both."
         },
         "parameters": {
-          "type": "object",
+          "$ref": "#/definitions/DeploymentParameter",
```

I don't think this is quite right - you're defining the type of "parameters" as if it were a single parameter. Instead, I think you want to define this as  a dictionary:

```suggestion
          "type": "object",
          "additionalProperties": {
            "$ref": "#/definitions/DeploymentParameter"
          },
```